netstandard/ref/System.Collections.Generic.cs

```diff
@@ -252,19 +270,25 @@ public partial interface ISet<T> : System.Collections.Generic.ICollection<T>, Sy
         void SymmetricExceptWith(System.Collections.Generic.IEnumerable<T> other);
         void UnionWith(System.Collections.Generic.IEnumerable<T> other);
     }
-    public partial class KeyNotFoundException : System.SystemException, System.Runtime.Serialization.ISerializable
+    public partial class KeyNotFoundException : System.SystemException
     {
         public KeyNotFoundException() { }
         protected KeyNotFoundException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public KeyNotFoundException(string message) { }
         public KeyNotFoundException(string message, System.Exception innerException) { }
     }
+    public static partial class KeyValuePair
+    {
+        public static System.Collections.Generic.KeyValuePair<TKey, TValue> Create<TKey, TValue>(TKey key, TValue value) { throw null; }
+    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public partial struct KeyValuePair<TKey, TValue>
     {
         public KeyValuePair(TKey key, TValue value) { throw null;}
         public TKey Key { get { throw null; } }
         public TValue Value { get { throw null; } }
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
```

Comment:
Do we need (want) to include the attribute in the NS ?

Reply:
I assume you are referring to the application of the attribute. In which case I do think we should. We decided to hide this from intellisense to avoid folks from calling it directly as this is mostly needed for the compiler. If your interested see the conversation at https://github.com/dotnet/corefx/issues/13746

Reply:
Agree with @weshaggard. We should have a consistent experience with these across all platforms.

Reply:
Do we also have consistent message that the attribute is not needed in the implementation? I think it can be confusing for someone implementing the standard whether it's needed to have these UI enhancements attributes in the implementation or not.

My preference should be exclude them from standard but have them as enhancements in the NS reference assemblies

Reply:
These attribute aren't necessary in the implementation but they also don't hurt anything being added there as well.

>My preference should be exclude them from standard but have them as enhancements in the NS reference assemblies

Currently the standard and the NS reference assemblies are one and the same thing. We could make a list of reference assembly only attributes that can be excluded from the implementations but we do likely want to keep them in the reference assemblies for each of the platforms to make them consistently hidden or not.